core/src/main/resources/schema/20151027185309_schema_changes_for_keystone_v3.sql

```diff
@@ -0,0 +1,30 @@
+-- // schema changes for keystone v3
+-- Migration SQL that makes the change goes here.
+ALTER TABLE credential ADD COLUMN keystoneversion character varying(255) DEFAULT 'cb-keystone-v2';
+
```

Comment:
this will affect all type of credential, but can only affect OpenStackCredential. Instead of default value please use the following sql after the alter tables:

UPDATE credential SET keystoneversion='cb-keystone-v2' WHERE dtype='OpenStackCredential';